pkg/scheduler/util/utils_test.go

```diff
@@ -207,3 +211,65 @@ func TestGetContainerPorts(t *testing.T) {
 		}
 	}
 }
+
+func TestGetPodFullName(t *testing.T) {
+	pod := &v1.Pod{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: "test",
+			Name:      "pod",
+		},
+	}
+	got := GetPodFullName(pod)
+	expected := fmt.Sprintf("%s_%s", pod.Name, pod.Namespace)
+	if got != expected {
+		t.Errorf("Got wrong full name, got %s, expected: %s", got, expected)
+	}
+}
+
+func newPriorityPodWithStartTime(priority int32, startTime time.Time) *v1.Pod {
+	return &v1.Pod{
+		Spec: v1.PodSpec{
+			Priority: &priority,
+		},
+		Status: v1.PodStatus{
+			StartTime: &metav1.Time{Time: startTime},
+		},
+	}
+}
+
+func TestGetEarliestPodStartTime(t *testing.T) {
+	currentTime := time.Now()
+	pod1 := newPriorityPodWithStartTime(1, currentTime)
+	pod2 := newPriorityPodWithStartTime(2, currentTime.Add(time.Second))
+	pod3 := newPriorityPodWithStartTime(2, currentTime)
```

Comment:
It would be better to give `pod3` a `StartTime` different from `pod1`, otherwise we could not figure out whether the result `startTime` in L253 is `pod1`'s or `pod3`'s StartTime.

Reply:
done.